src/components/reports/ReportForm.tsx

```diff
@@ -94,9 +98,9 @@ const ReportForm = () => {
   const ctx = useContext(FhirContext);
 
   const onSuccessfulSubmitHandler = (values: FormValues, actions: FormikHelpers<FormValues>) => {
-    const bundle = bundleRequest(values.patient);
+    const bundle = bundleRequest(values);
 
-    setResult(JSON.stringify(bundle, null, 2));
+    setResult(JSON.stringify(JSON.parse(bundle.body), null, 2));
```

Comment:
Yeah, basically wanted to have a pretty printed JSON shown rather than a minimal whitespace JSON, a little bit ridiculous I know! Though long term this will be removed